### ISSUE TITLE
make `qudt:unit` optional

### ISSUE THREAD

**Issue by Rdataflow** [zazuko/cube-link#159]
currently as per https://www.w3.org/TR/shacl/#EqualsConstraintComponent the result set for `qudt:hasUnit` need to be the same as for `qudt:unit`

the current implementation makes `qudt:unit` necessary
https://github.com/zazuko/cube-link/blob/5b6f089d042c564bad59d7317e74c2d49c5a62ee/validation/standalone-constraint-constraint.ttl#L68-L71

where it should be optional i.e. with
```shacl
        [
            sh:path sh:property ;
            sh:or (
                [
                    sh:path qudt:unit ;
                    sh:equals qudt:hasUnit ;
                ]
                [
                    sh:path qudt:unit ;
                    sh:maxCount 0 ;
                ]
            ) ;
            sh:severity sh:Warning ;
            sh:message "The use of qudt:unit is deprecated, please use qudt:hasUnit instead" ;
        ] ;

```

cc @tpluscode

**Comment by tpluscode** [comment]
Is this related to the failures you reported in #158?

**Comment by Rdataflow** [comment]
It's not the root cause - but it happens to occurs related in some situations.

**Comment by Rdataflow** [comment]
Closed by #160
